### ISSUE TITLE
Full Accessibility & 508 Office Audit [Sitewide Content + Public Websites: Adding footer link]

### ISSUE THREAD

**Issue by DanielleThierryUSDSVA** [department-of-veterans-affairs/va.gov-team#23699]
We've added this footer link and would like to ask for an accessibility review. 

![image](https://user-images.githubusercontent.com/62957278/115893113-d937f380-a425-11eb-9903-dca81bd854e3.png)

Please let me know if you have any questions or need any information. Thank you!

Here's a link to the ticket for this work: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/23120

## Steps to complete the Full Accessibility & 508 Audit: 
- [ ] VFS: Verify all accessibility defects that were identified at Staging Review have been fixed before requesting a full audit *(VSP understands that other teams may be accountable for fixing the defects identified at Staging Review. These defects are exempt and will not block a team from scheduling this touchpoint)*
- [x] VFS Product Manager: Create this issue and fill in your team and feature name in the title
- [ ] VFS: Link to this issue once created in [#vfs-platform-support](https://dsva.slack.com/archives/CBU0KDSB1) in Slack; tag @ Trevor Pierce
- [ ] VSP: Create sub-issues attached to this epic with findings. Audits are performed on a first-come-first-serve basis and are scheduled with each biweekly sprint
- [ ] VFS: Coordinate with the VA 508 Office to address accessibility findings
- [ ] VSP: Update [Platform Collaboration Point Tracker](https://docs.google.com/spreadsheets/d/1OgPyEvUlNF6EnaYMFAXJkV6FKOvZnlPnbOQ2fAJ7W7A/edit#gid=1116695886)

## Artifacts
- Access information for the tool or feature in staging. Do not put staging credentials in your `va.gov-team` ticket; store or reference them in a .md file in the `va.gov-team-sensitive` repository
- Link to verification that accessibility checks are complete:
  - (preferred) Link to your [TestRail VSP accessibility test plan template](https://dsvavsp.testrail.io/index.php?/suites/view/14&group_by=cases:section_id&group_order=asc).
OR
  - Link to a doc in your product folder that indicates all required checks from the accessibility [staging review prep](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/staging-review-processes.md) doc are complete.
- List of known issues (with links to tickets when applicable)

## Additional information
As part of this review, VSP will send your product to VA's 508 office for approval (as applicable).

Please refer to the [Collaboration Cycle README](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/README.md) on Github for more information about the Collaboration Cycle.

**Comment by noahgelman** [comment]
@laurenernest @shiragoodman

No accessibility issues

**Comment by laflannery** [comment]
No issues were found, I do not think there is a need for this issue to remain open.  Will close in 1 week unless told otherwise.

**Comment by shiragoodman** [comment]
@laflannery I think it's safe for you to close this issue.
